modules/querying/partials/cardinality-api.adoc

```diff
@@ -0,0 +1,110 @@
+[#_compute_cardinality_statistics]
+=== Compute cardinality statistics
+
+`POST :14240/gsqlserver/gsql/stats/card`
+
+This endpoint computes the cardinality statistics for the vertex and edge types specified in the request.
+The cardinality statistics describe the number of vertices of a specified type, and the number of edges of a specified type between a specified pair of vertex types.
```

Comment:
One thing I forgot to mention - can you call out that this API is a preview feature and subject to change in the future? Just so that it's clear that these APIs should not be used in production.

Reply:
I added to the stats API page that all the APIs on the page are Preview